### ISSUE TITLE
Make large output collapsible in the log

### ISSUE THREAD

**Issue by curioussavage** [IceCreamYou/Chrome-BigConsole#7]
The output on the left is difficult to read if large objects are printed. 

Something like [this](https://github.com/chibicode/react-json-tree) might be a nice drop in replacement if you don't mind adding React to the project.

**Comment by IceCreamYou** [comment]
Yeah... the pretty printing could definitely be better. Part of the problem is that the hoops BigConsole has to jump through to be functional make it kind of awkward to do interactive output. So I can look more at that library but any solution is likely to need customization.
